website/static/_headers

```diff
@@ -3,16 +3,26 @@
 # They are safe for immutable caching, as filename change when content change
 
 /assets/*
-  cache-control: immutable
+  Cache-Control: public
+  Cache-Control: max-age=365000000
```

Comment:
These are not the static assets though :D So not sure you did what you originally wanted?